style.css

```diff
@@ -0,0 +1,67 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@200;300;400;500;600;700;800&family=Roboto&display=swap");
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.body {
+  font-family: 'Poppins', sans-serif;
```

Comment:
- [ ] The text font is not the same as the original design, of course, you have used `
Poppins, sans-serif` but in the design template it is `Popins` only, try changing this and see what will the design look like

<img width="623" alt="1" src="https://github.com/Kwamay/My-Porforlio/assets/77843050/c914e683-bdc6-4b25-b582-bca5e282ad43">